Formula/coq.rb

```diff
@@ -13,8 +13,8 @@ def message; <<-EOS.undent
 class Coq < Formula
   desc "Proof assistant for higher-order logic"
   homepage "https://coq.inria.fr/"
-  url "https://coq.inria.fr/distrib/V8.5/files/coq-8.5.tar.gz"
-  sha256 "89a92fb8b91e7cb0797d41c87cd13e4b63bee76c32a6dcc3d7c8055ca6a9ae3d"
+  url "https://coq.inria.fr/distrib/8.5pl1/files/coq-8.5pl1.tar.gz"
```

Comment:
This version isn't being detected correctly. unfortunately. Please add a dedicated `version "8.5pl1"` line. Thanks!